worker/uniter/manifold.go

```diff
@@ -89,6 +90,8 @@ func Manifold(config ManifoldConfig) dependency.Manifold {
 				return nil, errors.Errorf("expected a unit tag, got %v", tag)
 			}
 			uniterFacade := uniter.NewState(apiConn, unitTag)
+			modelConfigFacade := modelconfig.NewClient(apiConn)
```

Comment:
again, it should be a new API on the 'uniterFacade', not accessing the client facade.